src/php/iterators/php_htable_iterator.h

```diff
@@ -10,6 +10,7 @@ typedef struct ds_htable_iterator {
     uint32_t                 position;
     ds_htable_bucket_t      *bucket;
     ds_htable_t             *table;
+    zval                    *obj;
```

Comment:
The result will be the same, zend_object* is just a bit more compact. (Not that it matters here.)